app/cli.php

```diff
@@ -61,16 +61,16 @@
             return $dbForConsole;
         }
 
-        $dbAdapter = $pools
+        $connection = $pools
             ->get($project->getAttribute('database'))
-            ->pop()
-            ->getResource()
-        ;
+            ->pop();
+
+        $dbAdapter = $connection->getResource();
 
         $database = new Database($dbAdapter, $cache);
         $database->setNamespace('_' . $project->getInternalId());
 
-        return $database;
+        return [ $database, fn() => $connection->reclaim() ];
```

Comment:
This is the syntax we wanted to avoid with DB pools. We used to use it in realtime before. Lets not introduce it back again

Reply:
Updated.